pykern/pkinspect.py

```diff
@@ -70,6 +72,9 @@ def __str__(self):
             return "<no file>:0:<no func>"
 
 
+clsmembers = inspect.getmembers(sys.modules[__name__], inspect.isclass)
```

Comment:
This doesn't look like it is used anywhere. Also, if keeping it, should it be a const?

Reply:
d007b03 was a note to self...